### PR TITLE
Example scenario for Drupal

### PR DESCRIPTION
**Pull request by alexbuckgit**

### CODE REVIEW

File: docs/example-scenario/infrastructure/drupal-linux-azure.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+ms.date: 11/28/2018
+---
+
+# Deploy Drupal on Azure for greater operational resiliency
+
+Review this Azure example scenario to learn about deploying the Drupal content management system in Azure to achieve greater resiliency for content-based websites.
+
+This scenario is based on a solution built by a [microgrid](https://www.energy.gov/articles/how-microgrids-work) electricity producer and grid operator. As the company's business accelerated, customers increasingly relied on critical electricity services. The company realized it needed a more reliable cloud provider to support their business.
+
+As an early stage startup, the company could not afford a full-time software developer. They wanted a cloud provider with services usable by a tech-savvy end user. The founders decided to migrate to Azure and take advantage of Azure's platform as a service (PaaS) capabilities for their resilience and ease of use. The services chosen include Azure App Services for Linux, Azure DB for MySQL, Azure Application Gateway, Azure Container Registry, and Azure DevOps.
```
Comment (adamboeglin) [inline docs/example-scenario/infrastructure/drupal-linux-azure.md]: "They wanted a cloud provider with services usable by a tech-savvy end user. " sounds a bit too much like marketing

File: docs/example-scenario/infrastructure/drupal-linux-azure.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+* To scale your app services running Drupal, please see [Drupal server scaling](https://www.drupal.org/docs/8/managing-site-performance-and-scalability/server-scaling) best practices.
+* To maximize availablility, consider running your website in multiple geographic regions. For more information, see the reference architecture for [Running a web application in multiple Azure regions](/azure/architecture/reference-architectures/app-service-web-app/multi-region).
+* Using a cache such as [Azure Redis Cache](/azure/redis-cache) can significantly increase the performance of your website. For more information, see [Drupal integration with Redis](https://www.drupal.org/project/redis).
+* After creating a base Docker image and registering it in Azure Container Services, Drupal developers can easily pull the image to their local workstation, modify it, run it locally, or publish the changes to a development web server or app service slot. Because everything is Docker-based, you can be confident that your Drpual site will run the same on a dev laptop, a test server, or in production.
+
```
Comment (adamboeglin) [inline docs/example-scenario/infrastructure/drupal-linux-azure.md]: Needs a pricing section

File: docs/example-scenario/infrastructure/drupal-linux-azure.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+* Optimizing operations by modernizing websites running on Linux. By switching from infrastructure as a service (IaaS) to platform as a service (PaaS), you can lower your operational costs and increase the agility of your developer and operations staff. That agility accelerates your product's time-to-market and helps your organization respond quickly to changing marketplace and customer demands.
+* Empowering your employees with fast internal business apps, enabling them to provide better services to other departments and customers.
+* Better engaging with your customers via a fast and reliable customer portal.
+
+## Architecture
+
+![Architecture for Drupal on Azure][architecture]
+
+1. Web user requests are intially routed through an Azure Application Gateway. The gateway performs application-level (layer 7) load balancing, but in this case the gateway is used as a web application firewall (WAF). The WAF helps prevent common web server attacks such as SQL injection and cross-site scripting.
```
Comment (adamboeglin) [inline docs/example-scenario/infrastructure/drupal-linux-azure.md]: These are too wordy and explaining in too much detail.  The details of why something was done should be in considerations or components, this section should be an easy to follow flow.

File: docs/example-scenario/infrastructure/drupal-linux-azure.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+* Increased operational resilience by running Drupal on Azure App Services and Azure Database for MySQL instead of running on virtual machines, and by storing backups in separate geographic locations.
+* Significantly improved operational agility, driving innovations and changes to market quickly with fewer mistakes.
+* Increased security by using Azure Application Gateway’s web application firewall.
+* Optimized operations using Docker container technology for quick and accurate deployments.
+
+## Relevant use cases
+
+Other relevant use cases include:
+
+* Optimizing operations by modernizing websites running on Linux. By switching from infrastructure as a service (IaaS) to platform as a service (PaaS), you can lower your operational costs and increase the agility of your developer and operations staff. That agility accelerates your product's time-to-market and helps your organization respond quickly to changing marketplace and customer demands.
```
Comment (adamboeglin) [inline docs/example-scenario/infrastructure/drupal-linux-azure.md]: Are these three use cases?

**Comment by alexbuckgit** [comment]
Closed - scenario is on hold.
